app/models/hmcts_common_platform/court_application.rb

```diff
@@ -2,12 +2,12 @@ module HmctsCommonPlatform
   class CourtApplication
     attr_reader :data
 
+    delegate :blank?, to: :data
+
     def initialize(data)
       @data = data || {}
```

Comment:
Any reason this isn't `@data = HashWithIndifferentAccess.new(data || {})` ?

Reply:
there is a bad reason: I forgot this one :)